utils/release_helper.py

```diff
@@ -0,0 +1,788 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""
+Script created to help maintainers during the release process by automating Github tasks.
+"""
+
+# References:
+# - https://developer.github.com/v3/libraries/
+# - https://github.com/PyGithub/PyGithub
+# - https://pygithub.readthedocs.io/en/latest/github_objects.html
+# - https://docs.github.com/en/rest
+
+from datetime import datetime, timedelta
+from github import Github
+import argparse
+import configparser
+import os.path
+import re
+import subprocess
+
+
+def get_parameter_from_ini(config_file, section, parameter) -> str:
+    config = configparser.ConfigParser()
+    try:
+        config.read(config_file)
+        return config[section][parameter]
+    except Exception as e:
+        print(f'{e} entry was not found - Information cannot be retrieved!')
+        exit(1)
+
+
+def create_ini_file_template(creds_file):
+    try:
+        new_creds_file = os.path.expanduser(creds_file)
+        with open(new_creds_file, mode='w', encoding='utf-8') as file:
+            file.write(
+                '[DEFAULT]\n'
+                'github_token = <generate your token in https://github.com/settings/tokens>\n')
+        print(f'Great! {new_creds_file} was created! Edit it to include your personal token.')
+    except Exception as e:
+        print(f'Error: {e}')
+        exit(1)
```

Comment:
ditto, I find it unsettling that a function exits the whole program if something goes wrong.

Reply:
Done. I reviewed the places where the `exit(1)` was used.